### PR TITLE
Update react storybook node

### PR DESCRIPTION
**Pull request by kirtapnosslin**

### CODE REVIEW

File: src/components/avatar/Avatar.tsx [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
 import { colors } from "../../styles/constants"
 import * as types from "../../types"
 
 type P = Partial<AvatarProps>
 
-const StyledImg = styled.img`
+const StyledImg = styled.img<P>`
   border-radius: 50%;
-  height: ${(props: P) => props.width}px;
-  width: ${(props: P) => props.width}px;
```
Comment (Doverstav) [inline src/components/avatar/Avatar.tsx:12]: Re-add explicit typing

File: src/components/article/Article.tsx [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import React from "react"
-import styled from "styled-components"
+import { StyleSheetManager, styled } from "styled-components"
```
Comment (Doverstav) [inline src/components/article/Article.tsx]: Remove unused import

File: package.json [hunk 3]
```diff
@@ -15,86 +15,95 @@
...
-    "mapbox-gl": ">=1.13",
+    "mapbox-gl": ">=2.15",
     "normalize.css": ">=8.0",
     "polished": ">=4.2",
     "react": ">=17.0",
     "react-dom": ">=17.0",
-    "react-mapbox-gl": ">=5.1",
+    "react-map-gl": ">=7.1",
     "react-transition-group": ">=4.4",
     "styled-components": ">=5.3"
```
Comment (Doverstav) [inline package.json]: Should we bump this peer dependency to >=6.0.0, like with the optional dependency above?

Comment (Doverstav) [reply package.json]: "This" dependency being `styled-components`

Comment (kirtapnosslin) [reply package.json]: yes

**Review by Doverstav — APPROVED** [review]

**Pull request merged**